library/modules/es7.math.fscale.js

```diff
@@ -0,0 +1,40 @@
+// https://www.npmjs.com/package/ecma-proposal-math-extensions
+var $export = require('./_export')
+  // , scale = require('./es7.math.scale')
+  , f32A = new Float32Array(1);
```

Comment:
We can't use typed arrays here. It's not polyfilled in ES3 engines, in `library` version we haven't those global variables. You can use [those helpers](https://github.com/zloirock/core-js/blob/abb2b689cf16cd5aa67bebbb92ae5c1fa2a87a3d/modules/_typed-buffer.js#L39-L109), but need to extract it to separate file(s).